dapp/src/components/buySell/ApproveSwap.js

```diff
@@ -134,6 +137,13 @@ const ApproveSwap = ({
         'Contract approved'
       )
     }
+
+    if (!needsApproval) {
+      return ''
+    }
+    const route = `${routeConfig[needsApproval].name} to use your ${stableCoinToApprove.toUpperCase()}`
```

Comment:
moving route and routeMobile more immediately into the scope of the function where it is needed. Improves readability, because it is obvious that previous bits of the function don't need `route` and `routeMobile` info